config/more_samples-glyf_colr_1.py

```diff
@@ -683,6 +683,85 @@ def _composite(composite_mode, accessor_char):
     )
 
 
+def _foreground_color(fill_type, accessor_char):
+
+    fill_type_map = {
+        "solid": {
+            "Format": ot.PaintFormat.PaintSolid,
+            "PaletteIndex": 0xFFFF,
+            "Alpha": 1,
+        },
+        "linear": {
+            "Format": ot.PaintFormat.PaintLinearGradient,
+            "ColorLine": {
+                "ColorStop": [
+                    (0.0, *_cpal("white")),
+                    (0.5, 0xFFFF, 1),
+                    (1.0, *_cpal("white")),
+                ]
+            },
+            "x0": 100,
+            "y0": 250,
+            "x1": 900,
+            "y1": 250,
+            "x2": 100,
+            "y2": 300,
+        },
+        "radial": {
+            "Format": ot.PaintFormat.PaintRadialGradient,
+            "ColorLine": {
+                "ColorStop": [
+                    (0.0, *_cpal("white")),
+                    (0.5, 0xFFFF, 1),
+                    (1.0, *_cpal("white")),
+                ]
+            },
+            "x0": 500,
+            "y0": 600,
+            "r0": 50,
+            "x1": 500,
+            "y1": 600,
+            "r1": 450,
+        },
+        "sweep": {
+            "Format": ot.PaintFormat.PaintSweepGradient,
+            "ColorLine": {
+                "ColorStop": [
+                    (0.0, *_cpal("white")),
+                    (0.5, 0xFFFF, 1),
+                    (1.0, *_cpal("white")),
+                ]
+            },
+            "centerX": 500,
+            "centerY": 600,
+            "startAngle": -360,
+            "endAngle": 270,
+        },
+    }
+
+    if not fill_type in fill_type_map:
```

Comment:
the following is more pythonic:

```suggestion
    if fill_type not in fill_type_map:
```

btw, if fill_type really is not among the expected ones, I would let the KeyError propagate. There's no point in silently returning

Reply:
Removed this check.